src/generic/datavgen.cpp

```diff
@@ -2394,15 +2394,17 @@ void wxDataViewMainWindow::OnPaint( wxPaintEvent &WXUNUSED(event) )
 
                 if ( m_useCellFocus && m_currentCol && m_currentColSetByKeyboard )
                 {
-                    renderColumnFocus = true;
-
-                    // If this is container node without columns, render full-row focus:
-                    if ( !IsList() )
+                    // If this is an item with a single column, render full-row focus:
+                    numColsWithValue = 0;
```

Comment:
This is what seems to result in build errors on Travis CI, this variable is not declared.

Reply:
Thank you. Fixed.